src/features/multichain/components/CreateSafeOnNewChain/index.tsx

```diff
@@ -175,18 +198,21 @@ export const CreateSafeOnNewChain = ({
   )
 
   const safeCreationResult = useSafeCreationData(safeAddress, deployedChains)
-  const replayableChains = useReplayableNetworks(safeCreationResult[0], deployedChainIds)
+  const allCompatibleChains = useCompatibleNetworks(safeCreationResult[0])
+  const isUnsupportedSafeCreationVersion = Boolean(!allCompatibleChains?.length)
```

Comment:
This will also be true for zkSync Safes as they have no compatible networks.

Reply:
Also, I created a ticket for when there are no available chains to deploy to: https://www.notion.so/safe-global/76f265123c0a4ad680947ce489804342?v=f9f164524ec4464286fdc38f31589d0c&p=4ec720b334ea410bbe8abb083509bf7b&pm=s

Right now it just shows the network selector with no options

Reply:
Okay we can keep it as is then.